shell.nix

```diff
@@ -1,8 +1,9 @@
-{ pkgs ? import ./nix/nixpkgs.nix { enableMozillaOverlay = true; }
-, isDevelopmentShell ? true }:
+{ isDevelopmentShell ? true
```

Comment:
Perhaps you could add a comment that explains the meaning or effect of setting / unsetting `isDevelopmentShell`.

Reply:
Done, thanks.